### PR TITLE
add service monitor to tracing

### PR DESCRIPTION
**Pull request by rubenvp8510**
Signed-off-by: Ruben Vargas <ruben.vp8510@gmail.com>

### CODE REVIEW

File: configuration/examples/local/manifests/observatorium/tracing-servicemonitor-otel.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: otel-observatorium-xyz
+  namespace: observatorium
+spec:
+  endpoints:
+  - port: monitoring
```
Comment (pavolloffay) [inline configuration/examples/local/manifests/observatorium/tracing-servicemonitor-otel.yaml:8]: Do we enable metrics in the OTEL collector?

**Review by pavolloffay — APPROVED** [review]
LGTM, just one question.

File: configuration/examples/local/manifests/observatorium/tracing-servicemonitor-otel.yaml (hunk 1 above)
Comment (rubenvp8510) [reply configuration/examples/local/manifests/observatorium/tracing-servicemonitor-otel.yaml:8]: yes, and the operator creates the monitoring service. So I only need to add the ServiceMonitor.

**Comment by PhilipGough** [comment]
We can rebase on https://github.com/observatorium/observatorium/pull/495 when it merges which fixes the docs issue. Assuming ci isn't bricked :/

**Comment by rubenvp8510** [comment]
hello, can some one approve the CI for running? it need to be approved

" First-time contributors need a maintainer to approve running workflows. [Learn more.](https://docs.github.com/actions/managing-workflow-runs/approving-workflow-runs-from-public-forks)"

Thank you!!

**Comment by rubenvp8510** [comment]
cicirclie is still stuck :(

**Pull request closed without merging**